examples/storage_volume_templates.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.exceptions import HPOneViewException
+from config_loader import try_load_from_file
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+options = {
+    "name": "OneView Test Volume Template",
+    "provisioning":
+        {
+            "capacity": "235834383322",
+            "provisionType": "Thin",
+            "storagePoolUri": ""
+        },
+        "type": "StorageVolumeTemplateV3"
+}
+
+oneview_client = OneViewClient(config)
+
+# Find or add storage pool to use in template
+print("Find or add storage pool to use in template")
+storage_pools = oneview_client.storage_pools.get_all()
+storage_pool_added = False
+storage_system_added = False
+if storage_pools:
+    storage_pool = storage_pools[0]
+    print("   Found storage pool '{}' at uri: '{}".format(
+        storage_pool['name'], storage_pool['uri']))
```

Comment:
I think is more readable format the strings like this:   
`"   Found storage pool '{name}' at uri: '{uri}".format(**storage_pool)`